_build/builder/BuildDocs.py

```diff
@@ -73,27 +88,19 @@ def Setup(self):
 
     def DoTask(self):
         tmpDir = join(self.buildSetup.tmpDir, "chm")
-        Prepare()
-        #warnings.simplefilter('ignore', DeprecationWarning)
-        sphinx.build_main([
-            None,
-            # '-q',  # be quiet
-            "-a",  # always write all output files
-            "-b", "htmlhelp",
-            "-E",  # Don’t use a saved environment (the structure
-                   # caching all cross-references),
-            "-N",  # Prevent colored output.
-            "-P",  # (Useful for debugging only.) Run the Python debugger,
-                   # pdb, if an unhandled exception occurs while building.
-            "-D", "release=%s" % self.buildSetup.appVersion,
-            "-D", "templates_path=[]",
-            "-d", EncodePath(join(self.buildSetup.tmpDir, ".doctree")),
-            '-v', '-v',  # verbosity, can be given up to three times
-            # write warnings and errors to file
-            # '-w', join('output', 'sphinx_log.txt'),
-            EncodePath(DOCS_SOURCE_DIR),
-            tmpDir,
-        ])
+        WritePluginList(join(self.buildSetup.docsDir, "pluginlist.rst"))
```

Comment:
Oh yeah. and when i stated This can be changed in the future I guess. I was talking about this piece in general and not the whole thing about the authors. my bad on that one. i just read over my comment again and I kind of lost myself with that one as well. 

LOL

it's good to note it as we come across different bits of code that may have to be changed even if it is in the future. it. I do not know if there is a way to search through the comments for specific phrases. or maybe we could place a marker in the comment that way we have the ability when the time comes to make the changes in that specific area of code for the new addition. and or change. this is a good way to make sure we "remember" all of the things we need to modify.